src/storage/slab/slab.c

```diff
@@ -189,6 +189,9 @@ _slab_recreate_items(struct slab *slab)
             }
         } else if (it->in_freeq) {
             _slab_put_item_into_freeq(it, slab->id);
+        } else if ((it->klen ) && (it->vlen == 0)) {
```

Comment:
(it->klen ) -> (it->klen)

Reply:
I remove brackets to be inline with rest of the code